lib/internal/http2/core.js

```diff
@@ -823,6 +822,44 @@ class Http2Session extends EventEmitter {
     return settings;
   }
 
+  // Proxies for socket address-related getters so users don't
+  // ever need to reach through to the socket
+  get remoteAddress() {
```

Comment:
These need to be added to the docs in `/doc/api/http2.md`. Should be done in this PR but can be done in a separate one if necessary